assets/agenda/components/preview/coverage/CoverageActions.tsx

```diff
@@ -70,61 +91,99 @@ function CoverageActionsComponent({
                 </a>
             </ToolTip>
         );
-    } else if (coverage.workflow_status === WORKFLOW_STATUS.COMPLETED) {
-        const wireItem = coverage.delivery_id == null ? null : (wireItems || []).find(
-            (content) => content._id === coverage.delivery_id
-        );
-
-        if (wireItem != null && !(hideViewContentItems || []).includes(wireItem._id)) {
-            content.unshift(wireItem._access === true ? (
-                <ToolTip>
-                    <a
-                        key="contentLink"
-                        className='nh-button nh-button--small nh-button--tertiary'
-                        title={gettext('Open in new tab')}
-                        href={`/wire?item=${coverage.delivery_id}`}
-                        target={contentLinkTarget}
-                        onClick={(e) => {
-                            e.stopPropagation();
-                        }}
-                    >
-                        {gettext('View Content')}
-                    </a>
-                </ToolTip>
-            ) : (
-                <ToolTip>
-                    <a
-                        key="contentLink"
-                        className='nh-button nh-button--small nh-button--tertiary'
-                        href={viewContentEmail ? `mailto:${viewContentEmail}` : undefined}
-                        target={contentLinkTarget}
-                        title={viewContentEmail ?
-                            gettext(
-                                'You don’t have access to this content, please contact {{email}}',
-                                {email: viewContentEmail}
-                            ) :
-                            gettext('You don’t have access to this content')
-                        }
-                        onClick={(e) => {
-                            e.stopPropagation();
-                        }}
-                    >
-                        {gettext('View Content')}
-                    </a>
-                </ToolTip>
-            ));
-        }
     }
 
-    return (
-        <div className='coverage-item__row d-flex justify-content-end gap-2'>
-            {content}
+    const [wireListOpen, setWireListOpen] = useState(true);
+    const wireIds = coverage.workflow_status !== WORKFLOW_STATUS.COMPLETED ?
+        [] :
+        (coverage.deliveries || [])
+            .map((delivery) => delivery.delivery_id || '')
+            .filter((wireId) => wireId.length > 0 && !(hideViewContentItems || []).includes(wireId));
+    const wireVersions = (wireItems || []).filter((item) => wireIds.includes(item._id));
+
+    return wireVersions.length === 0 ? (
+        <div className="coverage-item__row d-flex justify-content-end gap-2">
+            {actionButtons}
+        </div>
+    ) : (
+        <div className={classNames(
+            'nh-collapsible-panel pt-0 nh-collapsible-panel--small coverage-item__extended-row',
```

Comment:
It'd be good to extract the UI code for collapsible panel into a separate component. I saw there is `CollapseBoxWithButton` already defined.